docker/ssl_analyze/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+cryptography = "*"
```

Comment:
Please remove the cryptography dependency and use demisto/crypto as a base image.

Reply:
Done